finite-supply.adoc

```diff
@@ -146,53 +146,53 @@ mempool—a small chance at a large amount can be worth more than a
 * A century is a _long_ time from the Bitcoin perspective. If it is
-  still around, it has probably evolved enormously.
-* If an overwhelming economic majority finds it attractive to change
+  still around, it will have probably evolved enormously.
+* If an overwhelming economic majority finds it convenient to change
   the rules and introduce for example a perpetual annual 0.1% or 1%
   monetary inflation, the supply of bitcoin will no longer be finite.
-* With zero block subsidy and an empty or almost empty mempool, things
+* With zero block subsidy and an empty or nearly empty mempool, things
   can become shaky due to fee sniping.
 
-Since the transition to a fee-only economy is so far in the future, it
-might be wise to not jump to conclusions and try to fix the potential
-issues. For example, Peter Todd thinks there's a risk that Bitcoin's
-security budget won't be enough in the future and argued for a small
+Since the transition to a fee-only block reward is so far in the future, it
+might be wise not to jump to conclusions and try to fix the potential
+issues while we can. For example, Peter Todd thinks there's an actual risk that Bitcoin's
+security budget won't be enough in the future, and consequently argues for a small
 perpetual inflation in Bitcoin. However, he also thinks it's not a
-good idea to discuss that at this time, and on the
-https://www.whatbitcoindid.com/podcast/peter-todd-on-the-essence-of-bitcoin[What
-Bitcoin Did podcast he said]:
+good idea to discuss such an issue at this time, as
+https://www.whatbitcoindid.com/podcast/peter-todd-on-the-essence-of-bitcoin[he said on the What
+Bitcoin Did podcast]:
 
-[quote,Peter Todd on security budget, What Bitcoin Did podcast]
+[quote,Peter Todd on security budget, What Bitcoin Did podcast (2019)]
 ____
 But, that’s a risk like 10, 20 years in the future. That is a very
 long time. And, by then, who the hell knows what the risks are?
 ____
 
 Perhaps we could think of Bitcoin as something organic. Imagine a
-small slowly growing oak plant. And that you had never seen a full
-grown tree in your life. Would it not be wise then, to restrain your
-control issues and not set all rules in advance on how this plant
+small, slowly-growing oak plant. Imagine also that you have never seen a fully
+grown tree in your life. Wouldn't it be wise then to restrain your
+control issues instead of setting in advance all the rules on how this plant
 should be allowed to evolve and grow?
 
 === Conclusion
 
-Whether the bitcoin supply will grow above 21 million we cannot say
-today and that is probably not so bad. Ensuring that the security
+Whether the bitcoin supply will grow past 21 million we cannot say
+today, and that is probably not so bad. Ensuring that the security
 budget remains high enough is crucial but not urgent. Let's have this
-discussion in 10-50 years when we know more. If it's still relevant.
+discussion in 10-50 years, when we'll know more. If it will still be relevant then.
```

Comment:
I prefer "If it's still relevant", but maybe "If it's still relevant then"?

Reply:
I had doubts myself here. Let's go back to the original, it sounds better than my edit